pages/sponsor/index.vue

```diff
@@ -115,6 +115,10 @@ export default {
 
 <style lang="postcss" scoped>
 .icon {
-    @apply w-16 md:w-24 h-16 md:h-24 bg-contain bg-no-repeat;
+    @apply w-24 md:w-24 h-24 md:h-24 bg-contain bg-no-repeat;
```

Comment:
Let's change the icon's width/height to meet the [design spec](https://www.figma.com/file/W3hdepQ7v7ZGYRP6QctBXH/PyCon-Taiwan-%E5%AE%98%E7%B6%B2%E8%A6%96%E8%A6%BA-2022?node-id=1670%3A17624).

```suggestion
    @apply bg-contain bg-no-repeat;
    width: 112px;
    height: 112px;
    @media (min-width: 768px) {
        width: 138px;
        height: 138px;
    }
```